requirements_demos.txt

```diff
@@ -0,0 +1,3 @@
+matplotlib>=3.0.2,<3.1.0
+tqdm>=4.32.1,<4.33.0
```

Comment:
This range of version for `tqdm` creates incompatibilities with `swifter`.

When doing `pip install fklearn` on a clean env, I get the following error:
`ERROR: swifter 0.293 has requirement tqdm>=4.33.0, but you'll have tqdm 4.32.2 which is incompatible.`

So probably it is a good idea to allow a higher range for `tqdm`.

As of today, the latest version of `tqdm` is [4.35.0](https://pypi.org/project/tqdm/). This solves the problem:
`tqdm>=4.32.1,<4.36.0`